cycle_for/task_03.py

```diff
@@ -0,0 +1,12 @@
+# Даны два целых числа A и В, A>B. Выведите все нечётные числа от A до B включительно, в порядке убывания. В этой
+# задаче можно обойтись без инструкции if.
+
+A = int(input())
+B = int(input())
+
+if A % 2 != 0:
```

Comment:
The instruction says you can solve it without `if`. Try that

Reply:
a = int(input())
b = int(input())
for i in range(a - (a + 1) % 2, b - b % 2, -2):
    print(i, end=' ')

Reply:
```
a = int(input())
b = int(input())
for i in range(a - (a + 1) % 2, b - b % 2, -2):
    print(i, end=' ')
```